bindings.cpp

```diff
@@ -2222,12 +2166,101 @@ bool DeRestPluginPrivate::sendConfigureReportingRequest(BindingTask &bt)
 
         return sendConfigureReportingRequest(bt, {rq});
     }
+    else if (bt.binding.clusterId == BASIC_CLUSTER_ID) {
+        Sensor *sensor = dynamic_cast<Sensor*>(bt.restNode);
+        if (!sensor)
+        {
+            return false;
+        }
+
+        if (sensor->modelId().startsWith(QLatin1String("RDM00"))) // Hue wall switch module
+        {
+            deCONZ::NumericUnion dummy;
+            dummy.u64 = 0;
+            // add device mode value if not already present
+            if (bt.restNode->getZclValue(BASIC_CLUSTER_ID, 0x0034, bt.binding.srcEndpoint).attributeId != 0x0034)
```

Comment:
A note for later PRs: it would be good to add some defines for the manufacturer specific Hue Basic Cluster attribute ids 0x0032, 0x0033, etc to get rid of some magic numbers.

Reply:
We use attribute IDs throughout the code, also for the standard attributes.  Happy to change that, but that would be a biggie.

Of course (?) they are defined in `general.xml`.  Would be nice if we can define/maintain them only once.